tms-service/src/config.ts

```diff
@@ -5,7 +5,7 @@ import { IConfig } from './interfaces';
 
 // Load .env file into process.env if it exists. This is convenient for running locally.
 dotenv({
-  path: path.resolve(__dirname, '../.env'),
+  path: path.resolve(__dirname, '../.env.template'),
```

Comment:
The .template should not be used here. The .template is what DevOps can use to populate whatever deployment necessary. The .env file is something that you generate on your machine based on the .template file, that is for Dev environments. dotenv is only used on developer machines.